src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -552,7 +552,7 @@
 
                 /* create the tab navigation for each group of tabs */
                 for (var i = 0; i < tabGroups.length; i++) {
-                    var tabs = tabGroups[i].querySelectorAll('.tab');
+                    var tabs = tabGroups[i].querySelectorAll(':scope > .tab');
```

Comment:
see the top comment of this file. Also; why is this need practically? Doesnt `tabGroups[i]` already defines the scope :S

Reply:
Got ya. Missed the comment sorry :)

>  however, I didn't want to edit more than I had to for this to work

same features across files should be kept in sync. 

Good to know here is; if the exception page occurs _outside_ the profiler (which has tabs) AND the profiler toolbar is active, then base_js from the profiler applies. Without the toolbar base_js from twig bundle applies.